### DISCUSSION TITLE
Keeping scss modules in /pages folder. Any disadvantages ?

### DISCUSSION THREAD

**Discussion by Cicada95** [vercel/next.js#39647] [Help] [answered]
Hi,
I'm keeping some of my page specific scss in the /pages folder. Is there any disadvantages to this ? 
<img width="431" alt="image" src="https://user-images.githubusercontent.com/20374864/184854647-013ae118-20df-4e0a-a0f7-c44eac75514a.png">

**Comment by bobarros** [comment]
Without looking much into it, at least you are increasing your building time, creating pages that are not necessary.
And is this a real app or an exercise? 

What is the gain that you see by doing this?

**Reply by Cicada95** [reply]
Keeping whole project styles near the pages, components where those styles are applied makes searching for styles in the project easier, keeps the project tidy. The same approach by our team was used using create-react-app where we kept styles near the routes. 

This is a real application btw, I don't see any real difference if keeping the styles near the pages. Could you explain more how this approach can increase build times and create pages that are not necessary?

**Comment by gaston-flores** [comment, marked as answer]
No, there's no real disadvantage. Next only considers `.js`, `.jsx`, `.ts`, or `.tsx` files to be pages. 

I think it looks a bit messy to have the styles there though. If you want to keep the styles next to the files that use them and keep the pages directory free of clutter, you can separate the pages into `pages` and `containers`. Pages import the containers, and styles are next to the containers in their own folders. [Here's an example.](https://codesandbox.io/s/next-containers-zj6fpp)
